src/components/TypesOfRecipeCards/CommentCard.jsx

```diff
@@ -0,0 +1,83 @@
+import Pizza from "../../assets/pizza.jpg";
+import User1 from "../../assets/user/User2.svg";
+
+const CommentCard = ({ user, review }) => {
+  //CommentCard grabs all the reviews created by the user
+  // user is the reviewer
+
+  const reviewContent = review ? review : null;
+
+  const orginalRecipe = reviewContent?.recipeId;
+
+  const reviewRating = reviewContent?.rating?.toFixed(1);
+  const recipeReview = reviewContent
+    ? reviewContent.userReview?.length > 100
+      ? reviewContent.userReview.slice(0, 105) + "..."
+      : reviewContent.userReview
+    : null;
```

Comment:
What are you trying to do here?

Reply:
the recipe review can be long, so I wanted to format it so each card is the same length/ same size. 
Later I can add a function to toggle and see the full review